concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -478,3 +484,19 @@ def test_blender_verification_order_should_call_verification_result_with_result_
             'error',
             ErrorCode.VERIFIER_COMPUTING_SSIM_FAILED.name,
         )
+
+    def test_that_blender_verification_order_should_return_false_if_sizes_in_pixels_are_not_equal(self):
```

Comment:
wrong test name

Reply:
Please write separate test case class if you are testing only this function. Or even better - a separate test file, because you will be really testing `utils.py`.

Reply:
done